DESCRIPTION

```diff
@@ -1,16 +1,22 @@
 Package: chipseq
 Title: chipseq: A package for analyzing chipseq data
 Version: 1.49.0
-Author: Deepayan Sarkar, Robert Gentleman, Michael Lawrence, Zizhen Yao
+Authors@R: c(
+        person("Deepayan", "Sarkar", role = "aut"),
+        person("Robert", "Gentleman", role = "aut"),
+        person("Michael", "Lawrence", role = "aut"),
+        person("Zizhen", "Yao", role = "aut"),
+        person("Oluwabukola", "Bamigbade", role = "ctb",
+               comment = "Converted vignette from Sweave to R Markdown / HTML."),
+        person("Bioconductor Package Maintainer",
+               email = "maintainer@bioconductor.org", role = "cre"))
 Description: Tools for helping process short read data for chipseq
-        experiments
+        experiments.
 Depends: R (>= 2.10), methods, BiocGenerics (>= 0.1.0), S4Vectors (>= 0.17.25),
         IRanges (>= 2.13.12), GenomicRanges (>= 1.31.8), ShortRead
-Imports: methods, stats, lattice, BiocGenerics, IRanges, GenomicRanges,
-        ShortRead
-Suggests: BSgenome, GenomicFeatures, TxDb.Mmusculus.UCSC.mm9.knownGene
-Maintainer: Bioconductor Package Maintainer
-        <maintainer@bioconductor.org>
+Imports: methods, stats, lattice, BiocGenerics, IRanges, GenomicRanges, ShortRead
+Suggests: BSgenome, GenomicFeatures, TxDb.Mmusculus.UCSC.mm9.knownGene, BiocStyle, knitr
```

Comment:
Could you break the long lines on 17 and 18 and indent the second line, for example, as it had been on old line 9 and 10 for Imports?